cohorts/functions.py

```diff
@@ -96,7 +96,7 @@ def neoantigen_count(row, cohort, filter_fn=None,
     return np.nan
 
 def expressed_missense_snv_count(row, cohort, **kwargs):
-    filter_fn = kwargs.pop("filter_fn")
+    filter_fn = kwargs.pop("filter_fn", None)
```

Comment:
Yup, I'm fairly certain that's correct. Did some double checking the other day, but should really have some tests.